modules/database-commons/src/main/java/org/testcontainers/ext/ScriptUtils.java

```diff
@@ -14,16 +14,14 @@
  * limitations under the License.
  */
 
-package org.testcontainers.jdbc.ext;
```

Comment:
this breaks public API

Reply:
ScriptUtils is mainly for internal usage (as it's stated in javadoc). No public testcontainers projects are using this class (I checked :) ). So personally I don't think it's a problem.

Reply:
I know at least one non-public project where it is used (I checked ;) ), so personally I think it is a problem :D

Reply:
So, some non-public, probably personal, project uses internal API of another project and therefore internal API of that project must be sealed in stone. The tail wagging the dog. 

Okay, I've kept public API of the Sql-based ScriptUtils - wrapper over new ScriptUtils.

Reply:
> probably personal

Your assumption is incorrect. These are some big companies actually ;)

Why do you think ScriptUtils is an internal API btw?

Reply:
>Why do you think ScriptUtils is an internal API btw?

It is said in the javadoc:
>* Generic utility methods for working with SQL scripts. Mainly for internal use
>* within the framework.